src/java/org/apache/cassandra/db/view/View.java

```diff
@@ -206,15 +206,20 @@ public ReadQuery getReadQuery()
 
     public synchronized void build()
     {
-        if (this.builder != null)
```

Comment:
I am not sure I understand correctly why you remove this check?

Reply:
The check is right below, in the `stopBuild()` method. I have just moved the entire `if` block to that separate method to be able to call it separately from `ColumnFamilyStore#truncateBlocking()` via `TableViews#stopBuild()`.